### PR TITLE
fix TestWALSegmentSizeOption for  windows.

### PR DESCRIPTION
**Pull request by krasi-georgiev**
Signed-off-by: Krasi Georgiev <kgeorgie@redhat.com>

### CODE REVIEW

File: db_test.go [hunk 1]
```diff
@@ -707,7 +707,9 @@ func TestWALSegmentSizeOption(t *testing.T) {
 		testutil.Ok(t, app.Commit())
 	}
 
-	files, err := ioutil.ReadDir(filepath.Join(db.Dir(), "wal"))
+	dbDir := db.Dir()
+	db.Close()
```
Comment (brian-brazil) [inline db_test.go]: This will close on every iteration, so I'd expect the 2nd iteration to fail.

Comment (krasi-georgiev) [reply db_test.go]: sorry I didn't understand. There is no loop here.

Comment (brian-brazil) [reply db_test.go]: Ah, indentation was showing oddly on the for above.

Comment (krasi-georgiev) [reply db_test.go]: yeah, I was just going to comment on that as well. It is all good the tests are passing

**Pull request merged**